src/components/Text/Text.tsx

```
@@ -0,0 +1,179 @@
+      sizeClasses = 'text-3xl';
+      break;
+    case '4xl':
+      sizeClasses = 'text-4xl';
+      break;
+    case '5xl':
+      sizeClasses = 'text-5xl';
+      break;
+    case '6xl':
+      sizeClasses = 'text-6xl';
+      break;
+    case 'md':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)